lib/Serialization/Serialization.cpp

```diff
@@ -4809,6 +4810,48 @@ void Serializer::writeToStream(raw_ostream &os, ModuleOrSourceFile DC,
   S.writeToStream(os);
 }
 
+void swift::serializeToBuffers(
```

Comment:
Can you implement `serialize` in terms of this, just so we don't have two implementations?

Reply:
Are we cool with double-buffering the serialization?

Reply:
Oh, right. It won't matter once we get a chance to do the mmap thing, but it does now.